public/less/_repository.less

```diff
@@ -1233,6 +1233,11 @@
                 }
             }
         }
+        .dlmenu {
+            position: absolute;
+            margin-left: -38px;
+            margin-top: -18px;
+        }
```

Comment:
This is rather ugly and such absolute positioned pixel hacks are bound to break eventually. Please find another way to layout the button correctly that does at least not rely on absolute positioning.